config/config.go

```diff
@@ -181,6 +188,15 @@ func (l *Local) validateAuthOfflineURL() error {
 	return nil
 }
 
+func (l *Local) showAuthWarning() {
+	switch l.IssueTracker {
+	case IssueTrackerGithub:
```

Comment:
Could you move this condition for the issue tracker being github in the if statement?
ie:
```
if l.IssueTracker == IssueTrackerGithub && l.Auth.Token == "" {
    fmt.Fprintln(color.Output, color.YellowString(githubAPITokenWarning))
}
```

This simplifies the changes to 3 LOC. Apart from that making the check in `showAuthWarning` is misleading because:
```
if l.Auth.Token == "" {
	l.showAuthWarning()
}
```

looks like the warning is always printed if the auth token is missing, which is not correct. The reader has to look at the `showAuthWarning` details to understand the whole picture.

The change with `color.Output` is done because otherwise, the colored output won't render properly in a windows terminal.

Reply:
Gotcha, that makes sense! I initially wanted it to be expandable but it makes more sense at this point to focus in on Github like you mentioned.